client/templates/meetingseries/meetingSeriesDetails.js

```diff
@@ -104,6 +105,30 @@ Template.meetingSeriesDetails.helpers({
 
     isModerator: function () {
         return isModerator();
+    },
+
+    isMeetingSeriesEdited: function () {
```

Comment:
This can be much simpler:
`return ms.isEditedBy !== Meteor.userId()`

`ms.isEditedBy` is equal to `undefined` or `null` if and only if `ms.isEditedDate` is equal to `undefined` or `null`. So we don't need to explicitly check `ms.isEditedDate`. That leaves us with the check that `ms.isEditedBy` is not set to the current user.

The method name should also reflect that fact, so it should probably be called `isMeetingSeriesEditedByAnotherUser`.

Reply:
Renaming "isMeetingSeriesEdited": I named it on purpose because the warning is also shown to the same user if he/she is working in multiple tabs or browser